ci/pipeline.yml

```diff
@@ -1,10 +1,20 @@
 ---
 jobs:
-- name: test
+
+- name: set-self
   plan:
-  - in_parallel:
     - get: secureproxy-release-git-repo
       trigger: true
+    - set_pipeline: self
+      file: config/ci/pipeline.yml
+      var_files:
+        - config/ci/config.yml
```

Comment:
It looks like this repo doesn't have a config.yml, and I don't see one in this PR